components/BottomNavigation/docs/accessibility.md

```diff
@@ -1,37 +1,48 @@
-### Color Theming
```

Comment:
This doesn't look like a "white space" change for this commit. What happened to the "Color Theming" section?